portfolio/src/main/webapp/tab.js

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+import {removeClassName, setToNone} from "./utils.js";
+
+class Tab {
+  constructor (){};
+
+  /**  
+   * Sets the default-tab by "clicking" on the "Background" tab, 
+   * which has id="default-tab.
+   */
+  displayDefaultSection = () => {
```

Comment:
Here and elsewhere, drop arrow functions for class methods. e.g ("displayDefaultSection() {..}"). Leave arrow functions for module-local functions that aren't class methods and any potential callback lamdba functions you might have.